src/Framework/TestCase.php

```diff
@@ -70,6 +66,12 @@ public function run($method = NULL)
 	 */
 	public function runTest($method, array $args = NULL)
 	{
+		if (!method_exists($this, $method)) {
+			throw new TestCaseException("Method '$method' does not exist.");
+		} elseif (!preg_match(self::METHOD_PATTERN, $method)) {
+			throw new TestCaseException("Method '$method' is not a testing method.");
```

Comment:
This forces `runTest()` usage only for test methods. IMHO OK, any other public method can be called directly. But maybe, there can be some use case for it.